{"name": "TC1_004: Signup_with_invalid_data via API", "status": "passed", "steps": [{"name": "Step 1: Submit POST request with invalid data.", "status": "passed", "start": 1737708117579, "stop": 1737708117579}, {"name": "Step 2: Verify failed Signup.", "status": "passed", "attachments": [{"name": "API Response", "source": "66b59db5-5496-492d-a4c9-2f759a3616bb-attachment.json", "type": "application/json"}], "start": 1737708117579, "stop": 1737708118604}], "attachments": [{"name": "log", "source": "def2165c-27e0-404e-b46c-e8372e5fe001-attachment.txt", "type": "text/plain"}], "start": 1737708117579, "stop": 1737708118605, "uuid": "f1d02acc-0a7b-45cf-8f63-9f826c503272", "historyId": "f1c067eb43dbb2d5ba79be6986a47ccf", "testCaseId": "f1c067eb43dbb2d5ba79be6986a47ccf", "fullName": "tests.api.test_signup_via_api.TestUserSignupFunctionality#test_failed_signup_using_invalid_data", "labels": [{"name": "suite", "value": "User SignUp Functionality"}, {"name": "subSuite", "value": "TC1_004: Signup_with_invalid_data via API"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "Regression"}, {"name": "feature", "value": "User SignUp Functionality"}, {"name": "tag", "value": "tags('api_signup_functionality')"}, {"name": "host", "value": "CHEQ-J7M1HB3"}, {"name": "thread", "value": "3652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_signup_via_api"}]}